pkg/provider/azure_standard.go

```diff
@@ -69,31 +69,31 @@ func (az *Cloud) getAvailabilitySetID(resourceGroup, availabilitySetName string)
 }
 
 // returns the full identifier of a loadbalancer frontendipconfiguration.
-func (az *Cloud) getFrontendIPConfigID(lbName, rgName, fipConfigName string) string {
+func (az *Cloud) getFrontendIPConfigID(lbName, fipConfigName string) string {
 	return fmt.Sprintf(
 		consts.FrontendIPConfigIDTemplate,
 		az.getNetworkResourceSubscriptionID(),
-		rgName,
+		az.getLoadBalancerResourceGroup(),
```

Comment:
getFrontendIPConfigID will rely on az config, the function will return different value if the config is changed.

Reply:
RG comes from `az.getLoadBalancerResourceGroup()` in all current usages. Added a new feature `XXXWithRG()` to allow using specific RG.
Updated.

Reply:
LGTM